spec/pgmoon_ssl_spec.moon

```diff
@@ -29,7 +29,7 @@ describe "pgmoon with server", ->
     assert pg\query "select * from information_schema.tables"
     pg\disconnect!
 
-  it "connects with ssl on ssl server (defaults to TLS v1.2)", ->
+  it "connects with ssl on ssl server (defaults to TLS v1.3)", ->
```

Comment:
Why are we bumping the TLS version for tests?

Reply:
Dependency changed. TLS v1 not supported Now.

Make Github actions pass.